onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.cpp

```diff
@@ -650,10 +659,28 @@ namespace Dml
         return false;
     }
 
+    bool IsCustomOpShader(const onnxruntime::Node& node)
+    {
+        auto custom_ops = std::array<char*, 2>{
```

Comment:
Perhaps add a comment to the STFT/DFT kernels to remove these lines if/when those custom ops are deleted.